### ISSUE TITLE
IControllerMetaData incorrect fields nullability

### ISSUE THREAD

**Issue by i4004** [SimplifyNet/Simplify.Web#97]
For example:

Now
```csharp
ControllerSecurity Security { get; }
```
Should be
```csharp
ControllerSecurity? Security { get; }
```

**Comment by muhmuhhum** [comment]
if still open i would try to fix this

**Comment by i4004** [comment]
@muhmuhhum, yes, feel free to create a PR.

**Comment by muhmuhhum** [comment]
Fixes #101
